template/README__labs.md

```diff
@@ -64,9 +63,8 @@ Since the legacy bundle includes many polyfills that are not needed by modern br
 
 Default [namespace]-[name]
 
-```html
-<[namespace]-[name]>Hello World</[namespace]-[name]>
-```
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./apiExamples/basic.md) -->
```

Comment:
```suggestion
<!-- AURO-GENERATED-CONTENT:START (CODE:src=./demo/apiExamples/basic.html) -->
```

Reply:
See below comment for why I don't want to move the apiExamples